### DISCUSSION TITLE
Is Kaniko dead?

### DISCUSSION THREAD

**Discussion by beth-soptim** [GoogleContainerTools/kaniko#3316] [General]
Hi,

was Kaniko abandoned by Google?

- apparently no maintainer (?)
- no bug triage in the last months
- no commits in the last months

**Comment by cniry** [comment]
Could you please respond, @JeromeJu , @QuanZhang-William ?

**Reply by QuanZhang-William** [reply]
Hi @cniry. @JeromeJu and I have stepped out of the project. I've forwarded this thread to the team that works on the project.

**Reply by cniry** [reply]
Oh, that's a shame you left the Kaniko project @QuanZhang-William, @JeromeJu, @aaron-prindle . it's a great tool. Thank you for your work. I hope the new team will keep it alive.

**Reply by lynomet** [reply]
@QuanZhang-William, @JeromeJu, @aaron-prindle  who can pick up and approves MRs ?

**Reply by abitrolly** [reply]
@QuanZhang-William @JeromeJu @aaron-prindle is there any interest to try `kaniko` as a testbed to see if crypto can save the maintenance of this project? https://tea.xyz/ and https://www.gitcoin.co/ are trying to make an impact in this area, but without trying it is hard to say if these can really cover any time spent by maintainers to address issues.
